scripts/git/bazel_to_cmake.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Add info + sample usage comment at top of file please (for each script).

Important info to include:

* `gh` install / documentation link
* mention that the scripts may force push to branches on the configured remote
* mention that certain branches must be pristine, and the scripts will `git checkout` as necessary (commit/stash any work before running them)